collectors/channels_collector.go

```diff
@@ -371,6 +377,29 @@ func (c *ChannelsCollector) Collect(ch chan<- prometheus.Metric) {
 		"waiting_close",
 	)
 
+	// Get the list of closed channels.
+	closedChannelsResp, err := c.lnd.ClosedChannels(context.Background())
+	if err != nil {
+		c.errChan <- fmt.Errorf("ChannelsCollector ClosedChannels "+
+			"failed with: %v", err)
+		return
+	}
+	closeCounts := make(map[string]int)
+	for _, channel := range closedChannelsResp {
+		typeString, ok := closeTypeLabelMap[channel.CloseType]
+		if !ok {
+			continue
```

Comment:
log an error since we really don't expect this?

Reply:
added